R/cran-compliance.R

```diff
@@ -39,17 +39,16 @@ vendor_pkgs <- function(path = ".", quiet = FALSE, overwrite = NULL) {
   cargo_lock_fp <- file.path(src_dir, "Cargo.lock")
 
   if (!file.exists(cargo_lock_fp)) {
-    withr::with_dir(src_dir, {
-      update_res <- processx::run(
-        "cargo",
-        c(
-          "generate-lockfile",
-          "--manifest-path",
-          file.path(src_dir, "Cargo.toml")
-        ),
-        stderr_line_callback = stderr_line_callback
-      )
-    })
+    update_res <- processx::run(
+      wd = src_dir,
+      "cargo",
+      c(
+        "generate-lockfile",
+        "--manifest-path",
+        file.path(src_dir, "Cargo.toml")
```

Comment:
This looks strange to me TBH, how does path relative to `src_dir` interplay with setting `wd` to `src_dir`?

Reply:
I agree. As you can see, I modified it only gently.